argusclient/client.py

```diff
@@ -377,6 +377,14 @@ def get_user_dashboard(self, ownerName, dashboardName, shared=True):
             assert len(dashboards) == 1, "Expected a single dashboard as a result, but got: %s" % len(dashboards)
             return dashboards[0]
 
+    def get_user_dashboards(self, ownerName, shared=True, limit=None, version=None):
+        """
+        Gets dashboards owned by ownerName.
+
+        :return: a list of :class:`argusclient.model.Dashboard` objects with all fields populated.
+        """
+        assert ownerName, "Expected a owner name"
+        return self.argus._request("get", "dashboards", params=dict(ownerName=ownerName, shared=shared, limit=limit, version=version))
```

Comment:
Shouldn't this be `owner`?
```suggestion
        return self.argus._request("get", "dashboards", params=dict(owner=ownerName, shared=shared, limit=limit, version=version))
```

Reply:
Oh, yes, good catch!